### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by DollyHernandez**
Hi Holmes. I updated “setup cypress configuration” and "first test" and "typescript test" and "static code"

### CODE REVIEW

File: cypress/integration/google.spec.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+describe("This is my first cypress test", () => {
+    it("should have a title", () => {
+        cy.visit("https://www.google.com/");
+        cy.title().should("eq", "Google");
+    });
+});
```
Comment (holgiosalos) [inline cypress/integration/google.spec.ts]: EOF: This means that you should add a blank line (line without spaces or tabs) at the end of the file. 

Github tells you this by showing this icon:
image

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            "cypress"
+        ]
+    },
+    "include": [
+        "cypress/**/*.ts"
+    ],
+    "exclude": [
+        "node_modules/"
+    ]
+}
```
Comment (holgiosalos) [inline tsconfig.json]: EOF

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+              loader: 'ts-loader',
+              options: {
+                transpileOnly: true,
+              },
+            },
+          ],
+        },
+      ],
+    },
+  }
```
Comment (holgiosalos) [inline webpack.config.js]: EOF

File: cypress/integration/google.spec.ts (hunk 1 above)
Comment (DollyHernandez) [reply cypress/integration/google.spec.ts]: Hi Holmes, I update the file with tab

File: tsconfig.json (hunk 2 above)
Comment (DollyHernandez) [reply tsconfig.json]: Hi Holmes, I updated the file with tab (EOF)

File: webpack.config.js (hunk 3 above)
Comment (DollyHernandez) [reply webpack.config.js]: Hello Holmes, I updated the file with tab (EOF).
Thanks

File: cypress/integration/google.spec.ts [hunk 4]
```diff
@@ -3,4 +3,5 @@ describe("This is my first cypress test", () => {
         cy.visit("https://www.google.com/");
         cy.title().should("eq", "Google");
     });
-});
\ No newline at end of file
+}); 
+
```
Comment (holgiosalos) [inline cypress/integration/google.spec.ts]: Line 7 has some whitespaces, delete those whitespaces or tabs, without deleting line 7. Or maybe you have two blank lines, and it is just needed one.

File: tsconfig.json [hunk 5]
```diff
@@ -14,4 +14,5 @@
     "exclude": [
         "node_modules/"
     ]
-}
\ No newline at end of file
+}
+
```
Comment (holgiosalos) [inline tsconfig.json]: Line 7 has some whitespaces, delete those whitespaces or tabs, without deleting line 7. Or maybe you have two blank lines, and it is just needed one.

File: webpack.config.js [hunk 6]
```diff
@@ -21,4 +21,5 @@ module.exports = {
         },
       ],
     },
-  }
\ No newline at end of file
+  }
+
```
Comment (holgiosalos) [inline webpack.config.js]: Line 7 has some whitespaces, delete those whitespaces or tabs, without deleting line 7. Or maybe you have two blank lines, and it is just needed one.

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**